src/uwtools/tests/drivers/test_upp.py

```diff
@@ -160,10 +160,34 @@ def test_UPP_namelist_file_missing_base_file(caplog, driverobj):
     assert regex_logged(caplog, "missing.nml: State: Not Ready (external asset)")
 
 
-def test_UPP_output(driverobj):
-    with raises(UWNotImplementedError) as e:
+def test_UPP_output(driverobj, tmp_path):
+    fields = ["?"] * (UPP.NFIELDS - 1)
+    parameters = ["?"] * UPP.NPARAMS
+    # fmt: off
+    control_data = [
+        "2",                # number of blocks
+        "1",                # number variables in 2nd block
+        "2",                # number variables in 1st block
+        "FOO",              # identifier of 1st block
+        *fields,            # fields of 1st block
+        *(parameters * 2) , # variable parameters of 1st block
+        "BAR",              # identifier of 2nd block
+        *fields,            # fields of 2nd block
+        *parameters,        # variable parameters of 2nd block
+    ]
```

Comment:
This is a mock-up of the lines we'd find in a `postxconfig-NT.txt` file.

Reply:
The commenting of the data is particularly appreciated here